AutoTenez.py

```diff
@@ -275,9 +307,10 @@ def make_reservation(self, md5slotkey):
         # Parse input arguments
         parser = argparse.ArgumentParser(description='Reserve tennis court for tomorrow.')
         parser.add_argument('-c',  '--courts',               nargs='+',     help='Specify courts ("Baan X", where X is the court number). Default setting is all courts.')
-        parser.add_argument('-d',  '--date',                                help='Specify the date to make the reservation (yyyy-mm-dd). Defaults to `reservation_date`.')
+        parser.add_argument('-d',  '--date',                                help='Specify the date to make the reservation (yyyy-mm-dd). Defaults to tomorrow.')
         parser.add_argument('-t2', '--time_second_choice',   nargs='+',     help='Time you would like to reserve as a second option. One or two consecutive time slots separated by a space are allowed (hh:mm).')
         parser.add_argument('-c2', '--courts_second_choice', nargs='+',     help='Specify courts as second option ("Baan X", where X is the court number). Default setting is all courts.')
+        parser.add_argument('-q',  '--query',                               help='Query used when looking up the external reference of a member by name.')
```

Comment:
When using this argument, -t and -c are not required. However, that is pretty hard to include. Maybe add to known issues?

Reply:
I know, it's already there! This holds for the setup and query functionality, which shall both be moved to a separate file and inherit from the AutoTenez class.